inexor/engine/main.cpp

```diff
@@ -252,19 +211,12 @@ static void getbackgroundres(int &w, int &h)
     h = int(ceil(h*hk));
 }
 
-/// this function is called to render the main menu background once
-/// so the loading progress can be rendered over it continuously
-/// @see renderbackground
 void restorebackground()
 {
     if(renderedframe) return;
     renderbackground(backgroundcaption[0] ? backgroundcaption : NULL, backgroundmapshot, backgroundmapname[0] ? backgroundmapname : NULL, backgroundmapinfo, true);
 }
 
-
-
-/// Render a textured quad of the given dimensions.
```

Comment:
We shouldnt simply remove all doc comments but the ones which do not give any extra information.